### PR TITLE
make sure import module variable are unique

### PR DESCRIPTION
**Pull request by uittorio**
this is a quick fix until we understand completely how we are supposed to create unique variables names in typescript.

I need to create a test that proves unique variables.

NOTE: there are not really unique these variables, if someone use 'ɵRepository' 'ɵMerge' 'ɵExtensions' as variables name will have collissions issues

### CODE REVIEW

File: src/transformer/mockDefiner/uniqueIdentifier.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as ts from 'typescript';
+
+export function PrivateIdentifier(text: string): ts.Identifier {
+  return ts.createIdentifier(`ɵ${text}`);
```
Comment (Pmyl) [inline src/transformer/mockDefiner/uniqueIdentifier.ts]: don't we have other places where we use that character? could we reuse this function?

File: test/playground/playground.test.ts [hunk 2]
```diff
@@ -10,8 +10,8 @@ import { createMock } from 'ts-auto-mock';
 
 it('should work', () => {
   interface A {
-    a: string;
-  }
+  a: string;
```
Comment (Pmyl) [inline test/playground/playground.test.ts]: let's not touch this file as much as possible

File: src/transformer/mockDefiner/uniqueIdentifier.ts (hunk 1 above)
Comment (uittorio) [reply src/transformer/mockDefiner/uniqueIdentifier.ts]: done :)

File: test/playground/playground.test.ts (hunk 2 above)
Comment (uittorio) [reply test/playground/playground.test.ts]: done :)

**Comment by uittorio** [comment]
this should fix #210

**Pull request merged**